com.avaloq.tools.ddk.xtext.generator.test/src/com/avaloq/tools/ddk/xtext/generator/xbase/test/XbaseGeneratorFragmentTest.xtend

```diff
@@ -25,19 +24,27 @@ import org.eclipse.xtext.ParserRule
 import org.eclipse.xtext.RuleCall
 import org.eclipse.xtext.TypeRef
 import org.eclipse.xtext.XtextPackage
-import org.eclipse.xtext.testing.XtextRunner
-import org.junit.Test
-import org.junit.runner.RunWith
 
 import static org.mockito.Mockito.mock
 import static org.mockito.Mockito.when
 import org.eclipse.xtext.xtext.generator.xbase.XbaseUsageDetector
+import org.junit.jupiter.api.^extension.ExtendWith
+import org.eclipse.xtext.testing.extensions.InjectionExtension
+import org.junit.jupiter.api.Test
+import static org.junit.jupiter.api.Assertions.assertFalse
+import static org.junit.jupiter.api.Assertions.assertThrows
+import static org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.^extension.RegisterExtension
+import com.avaloq.tools.ddk.test.core.jupiter.BugTestAwareRule
 
 /**
  * Tests for {@link XbaseUsageDetector}.
  */
-@RunWith(XtextRunner)
-class XbaseGeneratorFragmentTest extends TestCase {
+@ExtendWith(InjectionExtension)
+class XbaseGeneratorFragmentTest {

```

Comment:
I actually have a question @rubenporras , this test was removed and is failing since forever. I tried to make it pass but im not successful. why does this test exist?